Tools/LyTestTools/tests/integ/test_o3de_codeowners.py

```diff
@@ -14,7 +14,7 @@
 def test_FindCodeowners_ThisCWD_FindForO3DE():
     codeowners_path = hint.find_github_codeowners(os.path.abspath(__file__))
     assert codeowners_path
-    assert "o3de" in str(codeowners_path), "codeowners file has been tampered with, or test unexpectedly migrated"
+    assert ".github" in str(codeowners_path), "codeowners file has been tampered with, or test unexpectedly migrated"
```

Comment:
Checking for .github in the path is essentially checking if the path exists which it already does above

Reply:
While the function above does currently assume that the path contains .github, this is not a requirement of a codeowners file, and this line in the test ensures that even if the above function changes that the codeowners file maintains its current location.